.github/CODEOWNERS

```diff
@@ -19,3 +19,4 @@
 /argocd/                  @ovotech/metering-industry-abstraction-cluster-admins
 /comms-commands/          @ovotech/comms
 /ipa-deploy/              @ovotech/ipa
+/sast-terraform/          @ovotech/consumer-products-production-engineering
```

Comment:
Our team is lacking write access to the repo which is causing this to display as an error

Reply:
Mentioned that the team needs this access in the PR description